docs/root/intro/arch_overview/http/http_connection_management.rst

```diff
@@ -169,15 +169,14 @@ For a redirect to be handled successfully it must pass the following checks:
    either 302 (by default), or a set of 3xx codes (301, 302, 303, 307, 308).
 2. Have a *location* header with a valid, fully qualified URL.
 3. The request must have been fully processed by Envoy.
-4. The request must not have a body.
```

Comment:
the request must have a body small enough Envoy is willing to buffer it?  I think it's mutable_per_request_buffer_limit_bytes for retries but I don't recall offhand if it's the same knob for redirects.

Reply:
updated docs

Reply:
generally we don't bother putting flags in non-release-note docs, so I think you can just assume the flag is true.